app/api/src/crud/crud_compute_heatmap.py

```diff
@@ -500,6 +502,48 @@ async def compute_opportunity_matrix(
                     poi_matrix[value],
                 )
 
+    async def get_neighbors(self, grids):
+        neighbors = set()
+        for h in grids:
+            neighbors_ = h3.k_ring(h, 1)
+            for n in neighbors_:
+                if not n in grids:
+                    neighbors.add(n)
+        return list(neighbors)
+
+    async def get_interior_neighbors(self, grids, study_area_polygon):
+        neighbors = await self.get_neighbors(grids)
+        neighbor_polygons = lambda hex_id: Polygon(h3.h3_to_geo_boundary(hex_id, geo_json=True))
+        neighbor_polygons = gpd.GeoSeries(list(map(neighbor_polygons, neighbors)), crs="EPSG:4326")
+        intersects = neighbor_polygons.intersects(study_area_polygon)
+        neighbors = list(compress(neighbors, intersects.values))
+        return neighbors
+
+    async def create_h3_girds(self, study_area_ids):
```

Comment:
there is type here. it should be create_h3_grids

Reply:
I should add a spell check to my vs-code :sweat_smile: